elements/pfe-clipboard/src/pfe-clipboard.js

```diff
@@ -91,12 +114,17 @@ class PfeClipboard extends PFElement {
     // the copy functionality
     this.addEventListener("click", this._clickHandler.bind(this));
     this.addEventListener("keydown", this._keydownHandler.bind(this));
+
+    // Make sure the thing we might copy exists
+    this.checkForCopyTarget();
   }
 
   disconnectedCallback() {
+    // Clean up after ourselves
     this.removeEventListener("click", this._clickHandler.bind(this));
     this.removeEventListener("keydown", this._keydownHandler.bind(this));
-    this.shadowRoot.removeEventListener("slotchange", this._slotchangeHandler.bind(this));
+    window.removeEventListener('DOMContentLoaded', this.checkForCopyTarget);
+    window.removeEventListener('load', this.checkForCopyTarget);
```

Comment:
@wesruv I think we're removing these right?

Reply:
D'OH!